src/throttle.js

```diff
@@ -0,0 +1,22 @@
+module.exports = function throttle (func, wait) {
+  var context, args, result, timeout, previous
+  var later = function () {
+    previous = Date.now()
+    result = func.apply(context, args)
+    timeout = context = args = null
+  }
+  return function () {
+    context = this
+    args = arguments
+    var now = Date.now()
```

Comment:
This will mean we can't use slapdash in old browsers anymore

Reply:
You are right, fixed

![](https://media1.giphy.com/media/l2QEbkmdsd7WNg5P2/giphy.gif?cid=5a38a5a23e318c59c233c12913a6431926cfb1e48e772f0e&rid=giphy.gif)